logs/fluent-bit/image/VERSION

```diff
@@ -1 +1 @@
-v0.0.3
+v2.1.3
```

Comment:
@MichaelBriggs-Coralogix sorry do we would like to switch to this versioning?

Reply:
Yes, that way our image version is similar to the fluentbit version just like we do in fluentd and it is less confusing to customers